www/artists/mbolen/gallery.html

```diff
@@ -0,0 +1,208 @@
+        <link href="css/stylish-portfolio.css" rel="stylesheet"><!-- Custom Fonts -->
+        <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+        <link href="css/onepage-scroll.css" rel="stylesheet" type="text/css">
+		<link href="css/galleryslideshow.css" rel="stylesheet" type="text/css">
+		<link href="css/styles.css" rel="stylesheet">
+        <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" rel="stylesheet" type="text/css">
+        <style type="text/css">
+        div.c9 {margin-top:10em;}
+        a.c8 { color: #050504;}
+        div.c7 {margin-top:8em;}
+        p.c6 {font-size:3em;padding-bottom:1em;}
+        p.c5 {font-family: 'Raleway', sans-serif; font-weight: 300;}
+        h1.c4 { padding-right:4em; font-family: 'Raleway', sans-serif; font-weight: 100;}
+        a.c3 { color: #ffffff;}
+        h1.c2 {font-family: 'Raleway', sans-serif; font-weight: 100;}
+        li.c1 {list-style: none; display: inline}
+        </style>
+<style>
+* {box-sizing: border-box}
+body {font-family: Verdana, sans-serif; margin:0}
+.mySlides {display: none}
+img {vertical-align: middle;}
+
+/* Slideshow container */
+.slideshow-container {
+  max-width: 1000px;
+  position: relative;
+  margin: auto;
+}
+
+/* Next & previous buttons */
+.prev, .next {
+  cursor: pointer;
+  position: absolute;
+  top: 50%;
+  width: auto;
+  padding: 16px;
+  margin-top: -22px;
+  color: white;
+  font-weight: bold;
+  font-size: 18px;
+  transition: 0.6s ease;
+  border-radius: 0 3px 3px 0;
+}
+
+/* Position the "next button" to the right */
+.next {
+  right: 0;
+  border-radius: 3px 0 0 3px;
+}
+
+/* On hover, add a black background color with a little bit see-through */
+.prev:hover, .next:hover {
+  background-color: rgba(0,0,0,0.8);
+}
+
+/* Caption text */
+.text {
+  color: #f2f2f2;
+  font-size: 15px;
+  padding: 8px 12px;
+  position: absolute;
+  bottom: 8px;
+  width: 100%;
+  text-align: center;
+}
+
+/* Number text (1/3 etc) */
+.numbertext {
+  color: #f2f2f2;
+  font-size: 12px;
+  padding: 8px 12px;
+  position: absolute;
+  top: 0;
+}
+
+/* The dots/bullets/indicators */
+.dot {
+  cursor: pointer;
+  height: 15px;
+  width: 15px;
+  margin: 0 2px;
+  background-color: #bbb;
+  border-radius: 50%;
+  display: inline-block;
+  transition: background-color 0.6s ease;
+}
+
+.active, .dot:hover {
+  background-color: #717171;
+}
+
+/* Fading animation */
+.fade {
+  -webkit-animation-name: fade;
+  -webkit-animation-duration: 9.0s;
+  animation-name: fade;
+  animation-duration: 9.0s;
+}
+
+@-webkit-keyframes fade {
+  from {opacity: .4} 
+  to {opacity: 1}
+}
+
+@keyframes fade {
+  from {opacity: .4} 
+  to {opacity: 1}
+}
+
+/* On smaller screens, decrease text size */
+@media only screen and (max-width: 300px) {
+  .prev, .next,.text {font-size: 11px}
+}
+</style>
+</head></style>
```

Comment:
What's going on here? Why are there two closing head tags, and no opening body tag? Was there a copy/paste mistake made here?

Reply:
@gemohandy What line is it on?

Reply:
Line 129.

Reply:
I dident know why that as there so I removed it.